.github/workflows/pr-check_markdownlint.yml

```
@@ -0,0 +1,41 @@
+name: Markdownlint (PR files)
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "**/*.md"
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v29.0.2
+        with:
+          files: |
+            **/*.md
+
+      - name: Copy upstream dependencies
+        run: |
+          curl -L https://raw.githubusercontent.com/mdn/content/main/package.json > package.json
```

```suggestion
          curl -L https://raw.githubusercontent.com/mdn/content/main/package.json -o package.json
```
file can be saved in the same command